Source/Details/ASDataController.mm

```diff
@@ -836,27 +820,3 @@ - (void)_scheduleBlockOnMainSerialQueue:(dispatch_block_t)block
 }
 
 @end
-
-#if AS_MEASURE_AVOIDED_DATACONTROLLER_WORK
```

Comment:
I see that this mechanism is valuable to you, but I truly believe this isn't the way to implement this. 

I spent some time this morning building a unit test to replace this. I failed, but ironically I discovered that this feature is currently _broken_ due to a subtle retain cycle where the node block retains the ASCollectionView. I've fixed that retain cycle, but I think it's worth noting that nobody noticed the feature was broken, possibly because this measurement feature is so obscure and unused. I care a lot about performance but I don't think this is the way to provide accountability.